src/riak_kv_wm_ts_util.erl

```diff
@@ -0,0 +1,121 @@
+%% -------------------------------------------------------------------
+%%
+%% riak_kv_wm_ts_util: utility functions for riak_kv_wm_timeseries* resources.
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(riak_kv_wm_ts_util).
+
+
+-export([table_from_request/1]).
+-export([utf8_to_binary/1]).
+-export([flat_format/2]).
+-export([set_text_resp_header/2]).
+-export([set_error_message/3]).
+-export([set_json_response/2]).
+
+-export([authorize/3]).
+
+-export([table_module_exists/1]).
+
+-export([local_key/1]).
+
+-export([local_key_fields_and_types/1]).
+
+-include_lib("webmachine/include/webmachine.hrl").
+-include_lib("riak_ql/include/riak_ql_ddl.hrl").
+-include("riak_kv_wm_raw.hrl").
+-include("riak_kv_ts.hrl").
+
+
+%% @private
+table_from_request(RD) ->
+    utf8_to_binary(
+      mochiweb_util:unquote(
+        wrq:path_info(table, RD))).
+
+%% move to util module.
+utf8_to_binary(S) ->
+    unicode:characters_to_binary(S, utf8, utf8).
+
+flat_format(Format, Args) ->
```

Comment:
This one-liner is too simple and too generic for an exported function. And, by placing it in riak_kv_wm_ts_util` you encourage callers in many other places in riak to call it from this module (otherwise, there is little sense in exporting it).